jms/src/main/java/com/sun/ts/tests/jms/core/bytesMsgQueue/BytesMsgQueueTestsEjbTest.java

```diff
@@ -78,19 +78,22 @@ public static EnterpriseArchive createDeploymentVehicle(@ArquillianResource Test
             com.sun.ts.lib.harness.EETest.class,
             com.sun.ts.lib.harness.ServiceEETest.class,
             com.sun.ts.lib.harness.EETest.SetupException.class,
-            com.sun.ts.tests.common.vehicle.VehicleClient.class
+            com.sun.ts.tests.common.vehicle.VehicleClient.class,
+            com.sun.ts.tests.jms.core.bytesMsgQueue.BytesMsgQueueTests.class
             );
             // The application-client.xml descriptor
             URL resURL = BytesMsgQueueTests.class.getResource("/com/sun/ts/tests/common/vehicle/ejb/ejb_vehicle_client.xml");
             if(resURL != null) {
               bytesMsgQueue_ejb_vehicle_client.addAsManifestResource(resURL, "application-client.xml");
             }
             // The sun-application-client.xml file need to be added or should this be in in the vendor Arquillian extension?
-            resURL = BytesMsgQueueTests.class.getResource("//com/sun/ts/tests/common/vehicle/ejb/ejb_vehicle_client.jar.sun-application-client.xml");
+            resURL = BytesMsgQueueTests.class.getResource("bytesMsgQueue_ejb_vehicle_client.jar.sun-application-client.xml");
             if(resURL != null) {
-              bytesMsgQueue_ejb_vehicle_client.addAsManifestResource(resURL, "application-client.xml");
+              bytesMsgQueue_ejb_vehicle_client.addAsManifestResource(resURL, "sun-application-client.xml");
             }
-            bytesMsgQueue_ejb_vehicle_client.addAsManifestResource(new StringAsset("Main-Class: " + BytesMsgQueueTests.class.getName() + "\n"), "MANIFEST.MF");
+            // bytesMsgQueue_ejb_vehicle_client.addAsManifestResource(new StringAsset("Main-Class: " + BytesMsgQueueTests.class.getName() + "\n"), "MANIFEST.MF");
+            bytesMsgQueue_ejb_vehicle_client.addAsManifestResource(new StringAsset("Main-Class: com.sun.ts.tests.common.vehicle.VehicleClient\n"), "MANIFEST.MF");
```

Comment:
@starksm64 The Main-class in MANIFEST file had to be replaced with VehicleClient to fix `javax.naming.NameNotFoundException`